tests/Unit/Time/Test_ChangeTimeStepperOrder.cpp

```diff
@@ -245,8 +270,175 @@ void test_falloff() {
+      tmpl::push_back<
+          time_stepper_ref_tags<LtsTimeStepper>,
+          ::Tags::IsUsingTimeSteppingErrorControlCompute<true>,
+          ::Tags::StepperErrorTolerancesCompute<System::variables_tag, true>>>(
+      Metavariables{}, std::move(time_stepper), VariableOrderAlgorithm(0.1),
+      initial_time_step_id,
+      time_stepper->next_time_id(initial_time_step_id, initial_time_step),
+      initial_time_step, slab.start().value(), ::AdaptiveSteppingDiagnostics{},
+      std::move(step_choosers), minimum_time_step, std::move(initial_vars),
+      dt_variables_tag::type{1}, history_tag::type{initial_order},
+      ::Tags::StepperErrors<System::variables_tag>::type{});
+
+  if (output_file.has_value()) {
+    ::file_system::rm(*output_file, false);
+  }
+
+  size_t order_sum = 0;
+  size_t num_steps = 0;
+
+  for (;;) {
+    if (output_file.has_value() and
+        db::get<::Tags::TimeStepId>(box).substep() == 0) {
+      Parallel::fprintf(
+          *output_file, "%zu\t%.16g\t%.16g\t%.16g\t%.16g\t%zu\n",
+          static_cast<size_t>(db::get<::Tags::AdaptiveSteppingDiagnostics>(box)
+                                  .number_of_steps),
+          db::get<::Tags::Time>(box), get(db::get<Var1>(box))[0],
+          get(db::get<Var2>(box))[0], db::get<::Tags::TimeStep>(box).value(),
+          static_cast<size_t>(db::get<history_tag>(box).integration_order()));
+    }
+    if (db::get<::Tags::TimeStepId>(box).slab_number() != 0) {
+      break;
+    }
+
+    order_sum += db::get<history_tag>(box).integration_order();
+    ++num_steps;
+
+    db::mutate_apply<System::compute_time_derivative>(make_not_null(&box));
+    take_step<System, true>(make_not_null(&box));
+    db::mutate_apply<ChangeTimeStepperOrder<System>>(make_not_null(&box));
+
+    db::mutate<::Tags::TimeStepId, ::Tags::Next<::Tags::TimeStepId>,
+               ::Tags::Time, ::Tags::AdaptiveSteppingDiagnostics, history_tag>(
+        [](const gsl::not_null<TimeStepId*> time_step_id,
+           const gsl::not_null<TimeStepId*> next_time_step_id,
+           const gsl::not_null<double*> time,
+           const gsl::not_null<AdaptiveSteppingDiagnostics*> diags,
+           const gsl::not_null<history_tag::type*> history,
+           const TimeStepper& local_time_stepper, const TimeDelta& time_step) {
+          if (next_time_step_id->substep() == 0) {
+            ++diags->number_of_steps;
+          }
+          *time_step_id = *next_time_step_id;
+          *next_time_step_id = local_time_stepper.next_time_id(
+              *next_time_step_id,
+              time_step.with_slab(time_step_id->step_time().slab()));
+          *time = time_step_id->substep_time();
+          local_time_stepper.clean_history(history);
+        },
+        make_not_null(&box), db::get<::Tags::TimeStepper<TimeStepper>>(box),
+        db::get<::Tags::TimeStep>(box));
+  }
+
+  return static_cast<double>(order_sum) / static_cast<double>(num_steps);
+}
+
+void test() {
```

Comment:
Could you add some documentation about this test? The wikipedia entry is not as insightful as I would like. E.g. what are the equations being solved and what makes them an interesting test case.